src/map/status.hpp

```diff
@@ -1031,9 +1031,9 @@ enum sc_type : int16 {
 	SC_WEAPONBLOCK_ON,
 	SC_SPORE_EXPLOSION,
 	SC_ADAPTATION,
-	SC_BASILICA_CELL, // Used in renewal mode for cell_basilica only
+	//SC_BASILICA_CELL,
```

Comment:
Maybe leave it uncommented and we remove it after the deprecation of script_constants.hpp has run out (2 years)?